staging/src/k8s.io/sample-controller/README.md

```diff
@@ -159,7 +159,7 @@ The CRD in [`crd-status-subresource.yaml`](./artifacts/examples/crd-status-subre
 for custom resources.
 This means that [`UpdateStatus`](./controller.go#L330) can be used by the controller to update only the status part of the custom resource.
 
-To understand why only the status part of the custom resource should be updated, please refer to the [Kubernetes API conventions](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+To understand why only the status part of the custom resource should be updated, please refer to the [Kubernetes API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
```

Comment:
https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status 302 redirects to `https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status` and we regularly use `git.k8s.io` links in our md files to link to documentation:

```
find . -name "*.md" | xargs egrep "git.k8s.io/community/contributors/devel" | wc -l
40
```

I don't think we should change this.

Reply:
> https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status 302 redirects to `https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status` and we regularly use `git.k8s.io` links in our md files to link to documentation:
> 
> ```
> find . -name "*.md" | xargs egrep "git.k8s.io/community/contributors/devel" | wc -l
> 40
> ```
> 
> I don't think we should change this.

Thanks Joe for your comments.

Do you know why not using the link directly? This confused me as the document is under github.com/kubernetes, why we use the git.k8s.io.  I guess this is a history issue that kubernetes is under git.k8s.io before, and some links are not changed.

Based on your input, I checked the usage of "github.com/kubernetes", there are 245 links, should we follow this? For the 40 links of "git.k8s.io", maybe we can change them all.

`find . -name "*.md" | xargs egrep "github.com/kubernetes" | wc -l
245`

Reply:
> Based on your input, I checked the usage of "github.com/kubernetes", there are 245 links, should we follow this? For the 40 links of "git.k8s.io", maybe we can change them all.
> 
> `find . -name "*.md" | xargs egrep "github.com/kubernetes" | wc -l 245`

That's not apples-to-apples. This is closer:

```
find . -name "*.md" | xargs egrep "github.com/kubernetes/community/blob/master/contributors/devel" | wc -l
9
```

Unfortunately, I don't know the history of the `git.k8s.io` links and don't have a strong opinion on if they're better or not. 

I agree that if we're going to do anything about the links (it's a really minor thing) then it's best to get consensus on the preferred way and update the whole code base to match.

Reply:
> > Based on your input, I checked the usage of "github.com/kubernetes", there are 245 links, should we follow this? For the 40 links of "git.k8s.io", maybe we can change them all.
> > `find . -name "*.md" | xargs egrep "github.com/kubernetes" | wc -l 245`
> 
> That's not apples-to-apples. This is closer:
> 
> ```
> find . -name "*.md" | xargs egrep "github.com/kubernetes/community/blob/master/contributors/devel" | wc -l
> 9
> ```
> 
> Unfortunately, I don't know the history of the `git.k8s.io` links and don't have a strong opinion on if they're better or not.
> 
> I agree that if we're going to do anything about the links (it's a really minor thing) then it's best to get consensus on the preferred way and update the whole code base to match.

Thanks for your comments, I will close this request now since the fix is minor and we don't know which is better, while the effor to fix all similar issues is big.